raiden/tests/integration/network/transport/rif_comms/test_client.py

```diff
@@ -1,252 +1,152 @@
+from typing import List, Dict
+
 import pytest
-from grpc._channel import _InactiveRpcError
 
 from raiden.tests.integration.network.transport.rif_comms.node import Node as CommsNode, Config as CommsConfig
 from raiden.tests.integration.network.transport.utils import generate_address
-from transport.rif_comms.utils import notification_to_payload, get_sender_from_notification
+from transport.rif_comms.client import Client
 
 
 @pytest.fixture()
-@pytest.mark.parametrize("amount_of_nodes")
-def comms_nodes(amount_of_nodes) -> {int, CommsNode}:
-    nodes = {}
-
+@pytest.mark.parametrize("cluster")
+def comms_clients(cluster: dict) -> Dict[int, Client]:
+    """
+    The cluster dict has the following structure:
+
+    {
+        1: 3 // a node with 3 clients connected
+        2: 1 // a node with a client
+        3: 2 // another node with 2 clients connected
+    }
+
+    Keys must be in increasing order from 1 to N
+    Values are the amount of clients connected to that node.
+    """
```

Comment:
added cluster class, not sure about keys.

Reply:
could you please give a more in-depth reply to the keys issue?

Reply:
"1" doesnt make sense to me, 1 is an integer. 

maybe "A", "B", "C" is better

Reply:
> "1" doesnt make sense to me, 1 is an integer.
> 
> maybe "A", "B", "C" is better

exactly. why are we using an integer in the place of a name instead of a quantity?

as values, integers make sense: we are going to create as many clients as the integer value dictates.

as for the cluster keys: they are merely names. `2` is not going to be used as an integer value under any circumstance in the current implementation. therefore, `"2"` is a more apt value, as it represents an (ordered) name.

as long as it is a string, i don't mind the value. if `A`, `B`, `C`, etc. is acceptable, then let's go with that.

fixtures would look something like `[{"A": 2, "B": 1}]`